pydeepmerge/strategies/_strategies.py

```diff
@@ -0,0 +1,40 @@
+from typing import Callable, Mapping, Union
+
+from pydeepmerge._deep_merge import MergeStrategy
```

Comment:
Added #3 to address this since I'd prefer to avoid files importing from the _files directly